chromium_src/chrome/browser/ui/webui/settings/browser_lifetime_handler.cc

```diff
@@ -22,10 +22,11 @@ BrowserLifetimeHandler::~BrowserLifetimeHandler() {}
 
 void BrowserLifetimeHandler::HandleRelaunch(const base::Value::List& args) {
 #if BUILDFLAG(ENABLE_SPARKLE) && !BUILDFLAG(BRAVE_ENABLE_UPDATER)
-  brave_relaunch_handler::RelaunchOnMac();
-#else
-  BrowserLifetimeHandler_ChromiumImpl::HandleRelaunch(args);
+  if (brave_relaunch_handler::RelaunchOnMac()) {
```

Comment:
is it okay that we ignore `args` here?

Reply:
@atuchin-m did you cross this out because your question is no longer relevant?

Reply:
yep, chromium also ignores `args`